### DISCUSSION TITLE
Unifi OS

### DISCUSSION THREAD

**Discussion by Ernie4President** [fabianishere/udm-iptv#238] [Q&A] [answered]
Last weekend I upgrade my UDM-pro to v2.4.27. All okay, except for the IPTV software. 
So, when I wanted to install IPTV, following the manual, I started terminal, logged in and tried to start Unifi OS. However, the message '-bash: unifi-os: command not found' is returned. Previously this worked fine. 
From other discussions ([https://github.com/fabianishere/udm-iptv/issues/212]) I learned installing the script should work, but couldn't find a way to get the software installed.

Can some help me? Am I missing something?

**Comment by fabianishere** [comment, marked as answer]
You don’t need to use the `unifi-os` command on v2.x. You can run the commands directly.

**Reply by Ernie4President** [reply]
Thanks @fabianishere ! Couldn't be more simple then this! Working like before 👍
